tests/test_web/test_rabbitmq.py

```diff
@@ -0,0 +1,115 @@
+from unittest import mock
+
+import pytest
+from kombu.exceptions import OperationalError
+
+
+def assert_testing(rv, mock_smfsc):
+    response_json = {'error': "Scheduling of request id 2 failed."}
+    assert rv.status_code == 500
+    assert response_json == rv.json
```

Comment:
It's less confusing if the same side is always picked for the actual value and the expected value
```suggestion
    assert rv.json == response_json
```